core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/runtimeRoutesModels.scala

```diff
@@ -103,6 +103,7 @@ final case class GetRuntimeResponse(id: Long,
                                     startUserScriptUri: Option[UserScriptPath],
                                     errors: List[RuntimeError],
                                     userJupyterExtensionConfig: Option[UserJupyterExtensionConfig],
+                                    autopause: Boolean,
```

Comment:
what is this boolean for?

Reply:
I removed the autopause column from the database, but kept it in the models so that it would keep the logic I previously implemented.

Reply:
@Qi77Qi I want the autopause value to be returned in the response for the UI, so shouldn't I keep this value here and not remove it? Or do you want me to refactor the UI side to work only with the threshold value?